stable/concourse/templates/web-deployment.yaml

```diff
@@ -51,6 +51,20 @@ spec:
                   name: {{ template "concourse.fullname" . }}
                   key: postgresql-uri
             {{ end }}
+            {{ if .Values.encryptionKey }}
```

Comment:
The image tag used in this chart is `2.6.0`. Looking at the https://hub.docker.com/r/concourse/concourse/tags/ page it seems this tag was create 7 months ago. Does this release have support for the encryption feature?